compiler/rustc_hir/src/hir.rs

```diff
@@ -3661,7 +3670,7 @@ mod size_asserts {
     use super::*;
     // tidy-alphabetical-start
     static_assert_size!(Block<'_>, 48);
-    static_assert_size!(Body<'_>, 32);
+    static_assert_size!(Body<'_>, 24);
```

Comment:
Yes, it makes `Body` smaller. I doubt that will have much of an effect, tho.

Reply:
Probably not, but it's better than making it bigger :)